menpo/base.py

```diff
@@ -210,9 +191,15 @@ def set_target(self, new_target):
         ----------
         new_target : :map:`PointCloud`
             The new target that this object should try and regenerate.
+
+        Returns
+        -------
+        self : :map:`Targetable`
+            A reference to this instance.
         """
         self._target_setter_with_verification(new_target)  # trigger the update
         self._sync_state_from_target()  # and a sync
+        return self
```

Comment:
Fair enough, but this behavior is inconsistent with the rest of our mutating API's  (from the top of my head `compose_{before,after}_inplace`, `from_vector_inplace`, `apply_inplace`) Saying that, I'm sure some classes do `return self` from those methods. It's an inconsistency I've been meaning to resolve for a while, guess we just need to decide which way to go.